lib/unison-sqlite/src/Unison/Sqlite/Sql.hs

```diff
@@ -0,0 +1,11 @@
+module Unison.Sqlite.Sql
+  ( Sql (..),
+  )
+where
+
+import Unison.Prelude
+
+-- | A SQL snippet.
+newtype Sql
+  = Sql Text
+  deriving newtype (IsString, Monoid, Semigroup, Show)
```

Comment:
Are Semigroup and Monoid reasonable instances here? Do we ever actually use it?
I guess a better answer is do we WANT to encourage them?

Also, do we want the `Derived` Show instance (it adds quotes right? or does it not?) or should we maybe return `show (Sql txt) = Text.unpack txt`

Reply:
I think we want to be able to concatenate fragments of queries together, yes.

I don't think the quotes in the derived show instance are an issue. It is a string, after all.

Reply:
I see, I think I would prefer that folks concatenate fragments as text then embed into Sql afterwards.

Seeing this instance implies at a glance that we can concatenate multiple sql statements or DDL, but that's not the case.

I don't feel super strongly about it though; it's pretty unlikely to come up 🤷🏼‍♂️

Reply:
In thinking about this more, the prepared statement cache proposal #2532 will at least need a type that represents a full Sql query, so if we want to compose Sql snippets together for query reusability, it will mean those queries can't be prepared.

Therefore I think it's a good idea to remove these instances after all.